### PR TITLE
Huihua/feature training review

### PR DESCRIPTION
**Pull request by ghh82058**

### CODE REVIEW

File: TMSFTT/urls.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 API_URLPATTERNS = [
     path('auth/', include('auth.urls')),
     path('training_event/', include('training_event.urls')),
+    path('training_review/', include('training_review.urls'))
```
Comment (Time1ess) [inline TMSFTT/urls.py]:
```suggestion
    path('training-review/', include('training_review.urls'))
```

File: training_review/tests/tests_views.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+User = get_user_model()
+
+
+class TestReviewNoteViewSet(APITestCase):
+    '''Unit tests for ReviewNote view.'''
+    def test_create_review_note(self):
+        '''ReviewNote should be created by POST request.'''
+        offcampusevent = mommy.make(tevent.OffCampusEvent)
```
Comment (Time1ess) [inline training_review/tests/tests_views.py]:
```suggestion
        off_campus_event = mommy.make(tevent.OffCampusEvent)
```

**Review by Time1ess — CHANGES_REQUESTED** [review]
Some duplicates are not commented, please fix them also.

**Review by Yqzzxlj — APPROVED** [review]

**Comment by Time1ess** [comment]
This branch is outdated, please fetch dev and rebase your branch against dev. @ghh82058

**Review by Time1ess — APPROVED** [review]

**Pull request merged**